_data/simple-icons.json

```
@@ -7095,6 +7095,11 @@
             "hex": "10B146",
             "source": "https://lmms.io/branding"
         },
+        {
+            "title": "Local",
+            "hex": "51BB7B",
+            "source": "https://localwp.com/wp-content/uploads/2021/08/favicon-1.png"
```

```suggestion
            "source": "https://localwp.com"
```
Do not link PNG. The icon is already available as SVG in the header of the front page.